content/en/logs/guide/lambda-logs-collection-troubleshooting-guide.md

```diff
@@ -0,0 +1,120 @@
+
+{{< img src="logs/guide/creating-trigger-error-example.png" alt="Example of error when creating trigger with subscription"  style="width:80%;" >}}
+
+See [Check for conflicting subscriptions](#check-for-conflicting-subscriptions) for more information on removing subscriptions.
+
+[1]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=allpermissions#manually-setup-triggers
+{{% /tab %}}
+{{% tab "Automatic trigger" %}}
+
+You can see if there are [automatic triggers][1] set up for your Lambda function with the following steps:
+
+1. Navigate to the [Cloudwatch console](https://console.aws.amazon.com/cloudwatch/).
+2. Click **Log Groups** in the left sidebar here you will see a list of Log Groups. There is a **subscriptions** column on the right that shows what resources (if any) are currently subscribed to the log source.
+3. If your Lambda function isn't listed as the subscriber for the Log Group you want to monitor, redo the steps from the [automatic trigger setup documentation][1].
+4. If the Log Group you want to monitor already has a different resource subscribed to it, see [Check for conflicting subscriptions](#check-for-conflicting-subscriptions) below.
+
+[1]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=allpermissions#automatically-setup-triggers
+{{% /tab %}}
+{{< /tabs >}}
+
+## Check the Lambda function logs
+
+From the monitoring tab, click **View logs in Cloudwatch**.
+
+{{< img src="logs/guide/lambda-logs-cloudwatch.png" alt="Lambda logs in Cloudwatch"  style="width:80%;" >}}
+
+### API key
+
+If you see one of the following log lines it means that your API key isn't set correctly.
+
+```
+module initialization error: Missing Datadog API key
+```
+This means you didn't set the API key. Places you can set the API key are:
+* Directly in the Python code of your Lambda function
+* As an environment variable under the name `DD_API_KEY`
+* As a KMS Encrypted key under the name `DD_KMS_API_KEY`
+
+```
+module initialization error: The API key is not valid.
+```
+The API key you provided doesn't match any key Datadog recognizes. Double-check the spelling and make sure that your key corresponds to the org you are sending data to. If your org is in Europe, you need to specify a `DD_SITE` variable with the value `datadoghq.eu` in order for the API key to be properly matched with your account.
+
+```
+module initialization error: The API key is not the expected length. Please confirm that your API key is correct
+```
+The API key is either too short or too long. Double-check that you copied it over correctly.
+
+## Check for conflicting subscriptions
+
+AWS doesn't allow for more than one resource to be subscribed to a log source. If you have triggers on your Lambda function but it still isn't being invoked, there may be a conflict with another resource already subscribed to the same log source. The log source needs to be freed up so the Datadog Lambda function can collect logs from it.
+
```

Comment:
A screenshot of a log group list that highlight the subscriber (last column on the right) could also be helpful to let users double check if there is not another function or service consuming the logs.